prow/cluster/starter.yaml

```diff
@@ -3,7 +3,7 @@
 # Source commit: 4faaf685958cd79ea5b5a376fadabd8a9d1b4123
 #
 # Changes:
-# - add get verb for configmaps resource in hook role
+# - move workload volume mapping for sinker/plank/deck to separate patch
```

Comment:
Here we only state the changes we make to the original `starter.yaml` file from kubernetes/test-infra.

Reply:
Restoring the old comment.